ndarray/src/main/java/org/tensorflow/ndarray/index/Indices.java

```diff
@@ -216,4 +255,129 @@ public static Index flip() {
   public static Index hyperslab(long start, long stride, long count, long block) {
     return new Hyperslab(start, stride, count, block);
   }
+
+  /**
+   * An index that inserts a new dimension of size 1 into the resulting array.
+   *
+   * @return index
+   */
+  public static TensorIndex newAxis(){
+    return NewAxis.INSTANCE;
+  }
+
+  /**
+   * An index that expands to fill all available source dimensions.
+   * Works the same as Python's {@code ...}.
+   * @see #expand()
+   * @return index
+   */
+  public static TensorIndex ellipsis(){
+    return Ellipsis.INSTANCE;
+  }
+
+  /**
+   * An index that expands to fill all available source dimensions.
+   * Works the same as Python's {@code ...}.
+   *
+   * @return index
+   */
+  public static TensorIndex expand(){
```

Comment:
why having both `expand()` and `ellipsis()` if they are the same?

Reply:
Because it really should be called "expand" or similar, but anyone coming from Python will be looking for ellipsis.  I could go with just one.